### PR TITLE
 Eboard transition: Follow style guide, update timeline, clarify off-boarding

### LINKED ISSUE #36: CampusGroups elections
From #34:
```
...A large section on elections is mostly missing, but will be revisited in the future.
```

### PR DESCRIPTION
**Pull request by jflory7**
## Review #39 first

This PR focuses on improving the eboard transition process. Specifically, it changes the following:

* Follow [style guide](https://documentation-style-guide-sphinx.readthedocs.io/en/latest/index.html) for Sphinx-based docs
* Clarifies election policy in context of CampusGroups (in a way, also closes #36)
* Emphasize a required step in off-boarding process (must be removed from CampusGroups)

---

_Tagging RITlug/tasks#8 for tracking purposes._

### CODE REVIEW

**Comment by ct-martin** [comment]
Also, Telegram

**Comment by jflory7** [comment]
@ct-martin Nice catch – I added in some instructions for Slack and Telegram in the on/off-boarding sections towards the bottom. Could you re-review?

File: docs/administration/eboard-onboarding-offboarding.rst [hunk 1]
```diff
@@ -18,82 +22,101 @@ Follow these steps to execute an election.
...
-^^^^^^^^^^^^^^^^^^^^^^^
+=======================
 
-- Grant operator privileges to incoming eboard members in the IRC channel (see how in :doc:`../infrastructure/irc-channel`).
+- Share Runbook with incoming eboard members, encourage questions and feedback on improving the Runbook
 
-- Add incoming eboard members as admins to `Facebook group`_
+- Incoming eboard members must take `financial certification test`_ from RIT clubs office (all eboard members **must be certified** to use club funds)
+
+- Add incoming eboard to private Slack channel for eboard discussion
```
Comment (ct-martin) [inline docs/administration/eboard-onboarding-offboarding.rst]: Shouldn't they also be given admin? As far as I understand, that's how we're doing things now.

Comment (jflory7) [reply docs/administration/eboard-onboarding-offboarding.rst]: @ct-martin Looking at how we do it now, that is how it's set up. I pushed a new commit to add an additional step.

**Pull request merged**